file.go

```diff
@@ -249,10 +257,8 @@ func (f *File) MarshallParts() (map[string]string, error) {
 	workbook = f.makeWorkbook()
 	sheetIndex := 1
 
-	if f.styles == nil {
-		f.styles = newXlsxStyleSheet(f.theme)
-	}
-	f.styles.reset()
```

Comment:
It worried me that the undocumented logic here was changed, but I can't find any reason why the style was reset when the style sheet was not new, so this is probably safe.